pkg/apply/apply.go

```diff
@@ -213,6 +218,11 @@ func (t *TopicApplier) applyExistingTopic(
 	}
 
 	if err := t.updatePartitions(ctx, topicInfo); err != nil {
+		if isFewerPartitionsError(err) && t.config.IgnoreFewerPartitionsError {
+			log.Warnf("UpdatePartitions failure ignored. topic: %v, error: %v", t.topicName, err)
+			return nil
+		}
```

Comment:
```suggestion
		if errors.Is(ErrFewerPartitions) && t.config.IgnoreFewerPartitionsError {
			log.Warnf("UpdatePartitions failure ignored. topic: %v, error: %v", t.topicName, err)
			
			return nil
		}
```

Reply:
FYI, the suggested code is missing an arg passed to the `errors.Is()`
It should be:
```
if errors.Is(err, ErrFewerPartitions) && t.config.IgnoreFewerPartitionsError {
	log.Warnf("UpdatePartitions failure ignored. topic: %v, error: %v", t.topicName, err)
	return nil
}
```

Reply:
Done.